packages/grafana-ui/src/components/Link/TextLink.story.tsx

```diff
@@ -0,0 +1,80 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+
+import { StoryExample } from '../../utils/storybook/StoryExample';
+import { VerticalGroup } from '../Layout/Layout';
+
+import { TextLink } from './TextLink';
+import mdx from './TextLink.mdx';
+
+const meta: Meta = {
+  title: 'General/TextLink',
+  component: TextLink,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+    controls: { exclude: ['href'] },
+  },
+  argTypes: {
+    variant: { control: 'select', options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'body', 'bodySmall', undefined] },
+    weight: {
+      control: 'select',
+      options: ['bold', 'medium', 'light', 'regular', undefined],
+    },
+    color: {
+      control: 'select',
+      options: [
+        'error',
+        'success',
+        'warning',
+        'info',
+        'primary',
+        'secondary',
+        'disabled',
+        'link',
+        'maxContrast',
+        undefined,
+      ],
+    },
+    externalLink: { control: 'boolean' },
+    inline: { control: 'boolean' },
+  },
+};
+
+export const Example: StoryFn = () => {
```

Comment:
I think these stories are reversed, the one with just 1 link should be the "Basic" and the one with more should be the "Examples" (and we could put more examples there like a combination of Inline with external and maybe put text around them to simplify how they would display in each of those cases?